src/index.ts

```diff
@@ -47,4 +47,7 @@ export { RecordsQuery, RecordsQueryOptions } from './interfaces/records-query.js
 export { RecordsRead, RecordsReadOptions } from './interfaces/records-read.js';
 export { SnapshotsCreate, SnapshotsCreateOptions } from './interfaces/snapshots-create.js';
 export { Secp256k1 } from './utils/secp256k1.js';
-export { SignatureInput } from './types/jws-types.js';
\ No newline at end of file
+export { SignatureInput } from './types/jws-types.js';
+export { DataStoreLevel } from './store/data-store-level.js';
+export { EventLogLevel } from './event-log/event-log-level.js';
+export { MessageStoreLevel } from './store/message-store-level.js';
```

Comment:
Need to update import paths in README https://github.com/TBD54566975/dwn-sdk-js#usage

Reply:
Thanks for the callout. Updated this PR with the following commit: https://github.com/TBD54566975/dwn-sdk-js/pull/477/commits/92f35bc05f76f953fd0aa6c6a2131ac4d3f62b40